src/main.cpp

```diff
@@ -162,15 +162,11 @@ int main(int argc, char *argv[]) {
     std::unique_ptr<EducationOptions> eduOptions (new EducationOptions(resourcePackManager));
     ServerInstanceEventCoordinator instanceEventCoordinator;
     std::unique_ptr<ServerInstance> instance (new ServerInstance(minecraftApp, instanceEventCoordinator));
-    LauncherV8Platform::initVtable(handle);
```

Comment:
https://github.com/ChristopherHX/mcpelauncher-server/blob/e36b10398e34427b8b2805c45723f111bf0d5d96/src/main.cpp#L165-L168
These Symbols are removed since 1.12, therefore crash the launcher

Reply:
Can you please comment this code out instead?

Reply:
Now only commented out